test/impute.test.ts

```diff
@@ -0,0 +1,69 @@
+/* tslint:disable:quotemark */
+import {assert} from 'chai';
+import {Encoding} from '../src/encoding';
+import {normalizeimputeProperties} from '../src/impute';
+
+describe('impute', () => {
```

Comment:
As I suggest that we remove `normalizeimputeProperties`, all of these tests should be moved to match the `test/compile/data/impute.test.ts`.